Markdown.Editor.js

```diff
@@ -1129,6 +1129,7 @@
 
             dialog.parentNode.removeChild(dialog);
 
+            inputBox.dispatchEvent(new Event('change'));

```

Comment:
```suggestion
            panels.input.dispatchEvent(new Event('change'));

```

esto funciona? aunque posiblemente sea mejor manejar esto en el callback o en el hook `imageConvertedToLink` y no aquí en este lugar específico.

Reply:
Parece que era suficiente el cambio en L811. Había estado buscando donde era el lugar indicado y por eso lo estaba poniendo en varios lados.